src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -0,0 +1,299 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { Button } from 'components/form/'
+import PDF from 'images/svg/pdf-icon-black.svg'
+import device from 'themes/device.js'
+import Chevron from 'images/svg/chevron-thick.svg'
+
+const StyledItemWrapper = styled(Flex)`
+    border-bottom: ${(props) => (props.is_expanded ? '1px solid var(--color-grey-8)' : 'none')};
+`
+const StyledIconWrapper = styled.div`
+    width: 100%;
+    max-width: 128px;
+    height: 80px;
+    margin-left: 16px;
+`
+const StyledChevron = styled.img`
+    height: 16px;
+    width: 16px;
+    margin: 26px 0 32px;
+    transform: ${(props) => (props.expanded ? 'inherit' : 'rotate(-180deg)')};
+    transition: transform 0.25s ease-out;
+`
+const HoverChevron = styled.div`
+    transition: background 0.25s;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-right: 16px;
+`
+const DetailsWrapper = styled(Flex)`
+    display: ${(props) => (props.is_expanded ? 'flex' : 'none')};
+`
+
+const StyledItemDiv = styled.div`
+    width: 50%;
+    margin-right: 8px;
+    @media ${device.mobileM} {
+        width: 130px;
+    }
+`
+const StyledKeyDiv = styled.div`
+    width: 50%;
+
+    @media ${device.mobileM} {
+        max-width: 120px;
+        display: flex;
+        justify-content: flex-start;
+    }
+`
+const StyledRow = styled(Flex)`
+    padding: 16px 18.5px;
+    border-bottom: 1px solid var(--color-grey-8);
+
+    :last-child {
+        border-bottom: none;
+    }
+`
+const BoldText = styled(Header)`
+    font-weight: bold;
+`
+const ValueText = styled(Header)`
+    text-align: right;
+
+    @media ${device.mobileM} {
+        text-align: left;
+    }
+`
+const RefIcon = styled.a`
+    display: flex;
+    justify-content: flex-end;
+`
+const StyledPDF = styled.img`
+    height: 32px;
+    width: 32px;
+`
+const StyledButton = styled(Button)`
+    margin-top: 8px;
+    max-width: 120px;
+`
+
+const MobileExpandedList = ({ is_crypto, is_fiat_onramp, is_dp2p, locale, payment_data }) => {
+    const [is_expanded, setIsExpanded] = React.useState(false)
+    const toggleExpand = () => {
+        setIsExpanded(!is_expanded)
+    }
```

Comment:
```suggestion
    const [is_expanded, setExpanded] = React.useState(false)
    const toggleExpand = () => {
        setExpanded(!is_expanded)
    }
```

Reply:
The usual naming convention is to drop the `is/has` for the setter function.

Reply:
Amended in latest commit..